zp-relayer/workers/sentTxWorker.ts

```diff
@@ -64,16 +64,17 @@ export async function createSentTxWorker<T extends EstimationType>(gasPrice: Gas
     for (let i = prevAttempts.length - 1; i >= 0; i--) {
       const txHash = prevAttempts[i][0]
       logger.info('Verifying %s ...', txHash)
-      tx = await web3.eth.getTransactionReceipt(txHash)
-      if (tx) break
+      try {
+        tx = await web3.eth.getTransactionReceipt(txHash)
+      } catch (e) {
+        logger.warn('Cannot get tx receipt for %s; Error: %s', txHash, (e as Error).message)
```

Comment:
should the `warn` log message really contain "Error" in the text?

Reply:
Changed to `Error` to `RPC response`